webCommander/centralHub/spotify/spotifyAPI.py

```diff
@@ -48,35 +51,56 @@ def get_spotify_artist(artist_name):
     Authenticate then search an artist based on an artist name. Searches for the name if id is not provided
     '''
 
-    if len(artist_name) != 22:
-        artist_id = search_spotify_artist(artist_name)
+    #TODO 
+    #Spotify artist is a 22 fixed character ID, but searching for a word would require to first call search_spotify_artist 
+    #to get the id which would also return some ids even if not an artist. Not sure how to handle this.
 
-    headers = spotify_authenticate()
+    if len(artist_name) != SPOTIFY_FIXED_ARTIST_CHAR_LEN:
+        artist_id = search_spotify_artist(artist_name)
 
     url = SPOTIFY_BASE_URL + SPOTIFY_ARTISTS + artist_id
-    response = requests.get(url, headers=headers)
+
+    response = requests.get(url, headers=spotify_authenticate())
     return response.text
 
 def search_spotify_artist(artist_name):
 
     url = SPOTIFY_BASE_URL + SPOTIFY_SEARCH
 
-    headers = spotify_authenticate()
-
     params = {
         "q": artist_name,
         "type": "artist"
     }
 
-    response = requests.get(url, headers=headers, params=params)
+    response = requests.get(url, headers=spotify_authenticate(), params=params)
     artists_results = json.loads(response.text)
 
     for artist in artists_results['artists']['items']:
         if artist['name'] == artist_name:
             return artist['id']
 
+def get_artist_albums(artist_name):
+    album_dict = {'images':[],
```

Comment:
you can use `collections.defaultdict` here, then you don't have to initialize the empty lists